pkg/asset/internal/templates.go

```diff
@@ -560,9 +600,12 @@ spec:
         resources:
           requests:
             cpu: 150m
-            memory: 10Mi
-      - name: dnsmasq-metrics
-        image: gcr.io/google_containers/dnsmasq-metrics-amd64:1.0
+            memory: 20Mi
+        volumeMounts:
+        - name: kube-dns-config
+          mountPath: /etc/k8s/dns/dnsmasq-nanny
```

Comment:
I think this is a mount to the configmap volume `kube-dns-config`.
Looks like the `dnsmasq nanny` is a replacement for the old `kube-dnsmasq-amd64:1.4` container